draft-ietf-dnsop-svcb-https.md

```diff
@@ -495,59 +495,58 @@ is the effective TargetName:
 
 # Client behavior {#client-behavior}
 
-A SVCB-aware client selects an endpoint for a service using the following
-procedure:
+"SVCB resolution" is the process of enumerating the priority-ordered endpoints
+for a service, as performed by the client.  It MUST be implemented as follows:
```

Comment:
```suggestion
for a service, as performed by the client.  SVCB resolution is implemented as follows:
```

I think that the MUST here is too strong.  The W3C and WHATWG, who specialize in writing algorithmic specifications, only ever say that the outcome should be equivalent to (or indistinguishable from) the algorithm as described.